[2024-04-20 13:27:50,546] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: NEO_coderhouse.Convertir_dataframe manual__2024-04-20T13:27:45.890819+00:00 [queued]>
[2024-04-20 13:27:50,573] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: NEO_coderhouse.Convertir_dataframe manual__2024-04-20T13:27:45.890819+00:00 [queued]>
[2024-04-20 13:27:50,574] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-04-20 13:27:50,576] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2024-04-20 13:27:50,577] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-04-20 13:27:50,619] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): Convertir_dataframe> on 2024-04-20 13:27:45.890819+00:00
[2024-04-20 13:27:50,628] {standard_task_runner.py:52} INFO - Started process 1442 to run task
[2024-04-20 13:27:50,637] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'NEO_coderhouse', 'Convertir_dataframe', 'manual__2024-04-20T13:27:45.890819+00:00', '--job-id', '149', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpje4r43sj', '--error-file', '/tmp/tmpxvw0gbcx']
[2024-04-20 13:27:50,640] {standard_task_runner.py:80} INFO - Job 149: Subtask Convertir_dataframe
[2024-04-20 13:27:50,802] {task_command.py:371} INFO - Running <TaskInstance: NEO_coderhouse.Convertir_dataframe manual__2024-04-20T13:27:45.890819+00:00 [running]> on host 2e237166a7ec
[2024-04-20 13:27:50,959] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=LukeFo
AIRFLOW_CTX_DAG_ID=NEO_coderhouse
AIRFLOW_CTX_TASK_ID=Convertir_dataframe
AIRFLOW_CTX_EXECUTION_DATE=2024-04-20T13:27:45.890819+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-04-20T13:27:45.890819+00:00
[2024-04-20 13:27:50,962] {logging_mixin.py:115} INFO - Hola
[2024-04-20 13:27:52,061] {logging_mixin.py:115} INFO - Cantidad de registros extraidos el 2024-04-20 - 1
[2024-04-20 13:27:52,064] {python.py:173} INFO - Done. Returned value was: {'neo_reference_id': {0: 3092476}, 'name': {0: '(2001 VE2)'}, 'absolute_magnitude_h': {0: 25.0}, 'is_potentially_hazardous_asteroid': {0: False}, 'is_sentry_object': {0: False}, 'estimated_diameter_min': {0: 26.58}, 'estimated_diameter_max': {0: 59.4346868419}, 'close_approach_date_full': {0: '2024-Apr-20 08:38'}, 'kilometers_per_second': {0: 14.2406261493}, 'distance_kilometers': {0: 48891486.144597575}, 'orbiting_body': {0: 'Earth'}}
[2024-04-20 13:27:52,139] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=NEO_coderhouse, task_id=Convertir_dataframe, execution_date=20240420T132745, start_date=20240420T132750, end_date=20240420T132752
[2024-04-20 13:27:52,223] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-04-20 13:27:52,321] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
